testing/array.test.ts

```diff
@@ -347,3 +347,15 @@ Deno.test("Array unzip", () => {
     [[1, 2, 3], ["a", "b", "c"]],
   );
 });
+
+Deno.test("Array sort", () => {
+  const t1 = [3, 1, 2];
+  const r1 = pipe(t1, A.sort(ordNumber));
+  assertEquals(r1, [1, 2, 3]);
+  assertEquals(t1, [3, 1, 2]);
+
+  const t2 = [1];
+  const r2 = pipe(t2, A.sort(ordNumber));
+  assertEquals(r2, t2);
+  assertNotStrictEquals(r2, t2);
```

Comment:
Could you add two test cases for:

1. An empty array.
2. An large, already ordered array (can be created using `range`).

Reply:
Please let me know if the empty array test fits your taste.

Reply:
Awesome